docsets/RxJava/docset.json

```diff
@@ -1,11 +1,24 @@
 {
     "name": "RxJava",
-    "version": "1.1.6",
-    "archive": "RxJava.tgz",
+    "version": "2.0.6",
+    "archive": "versions/2.0.6/RxJava.tgz",
```

Comment:
This doesn't work. The main docset archives and the specific version archive has to be separate.